cockpit/lib/cockpit-components-dialog.jsx

```diff
@@ -174,14 +174,18 @@ export class DialogFooter extends React.Component {
             actions_disabled = 'disabled';
```

Comment:
We generaly don't 'copy' lib but we auto fetch an tag using a Makefile target in all of the plugins we maintain. Not a must, but since you synced all these you could as well drop them and do the auto copying.

Reply:
Yeah that is something I would like to see in the future but right now will all the PF3, different build styles, no auto npm refreshes etc. it feels very disconnected from what other plugins do anyway. I would prefer to do that outside of this PR. Also I just copied two files.

Reply:
Sure, I can deal with that as well in the future.